README.md

```diff
@@ -1,15 +1,20 @@
-# Symfony CMF Seo Bundle
+# Symfony Cmf Seo Bundle
 
-[![Build Status](https://travis-ci.org/symfony-cmf/seo-bundle.svg?branch=master)](https://travis-ci.org/symfony-cmf/seo-bundle)
-[![StyleCI](https://styleci.io/repos/16372913/shield)](https://styleci.io/repos/16372913)
-[![Latest Stable Version](https://poser.pugx.org/symfony-cmf/seo-bundle/version.png)](https://packagist.org/packages/symfony-cmf/seo-bundle)
-[![Total Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/d/total.png)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![Latest Stable Version](https://poser.pugx.org/symfony-cmf/seo-bundle/v/stable)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![Latest Unstable Version](https://poser.pugx.org/symfony-cmf/seo-bundle/v/unstable)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![License](https://poser.pugx.org/symfony-cmf/seo-bundle/license)](https://packagist.org/packages/symfony-cmf/seo-bundle)
 
-This bundle is part of the [Symfony Content Management Framework (CMF)](http://cmf.symfony.com/)
-and licensed under the [MIT License](LICENSE).
+[![Total Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/downloads)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![Monthly Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/d/monthly)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![Daily Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/d/daily)](https://packagist.org/packages/symfony-cmf/seo-bundle)
 
-It provides a solution to make content bundles
-aware for Search Engine Optimisation (SEO).
+Branch | Travis | Coveralls |
+------ | ------ | --------- |
+2.0   | [![Build Status][travis_legacy_badge]][travis_legacy_link]     | [![Coverage Status][coveralls_legacy_badge]][coveralls_legacy_link]     |
+2.0   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
+master | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] |
+
+Symfony CMF Search Engine Optimization Bundle
 
 ## Requirements
```

Comment:
The requirements should be filled in from project configuration also. Doing so we got the same versions in readme as testing in travis.

Reply:
done.